graph_adventure/adv.py

```diff
@@ -187,12 +185,11 @@ def dft(world, startingRoom):
 
                 print(f"Moving '{go_back}' back to room {path[-1]}")
 
-                backtrack += 2
```

Comment:
your use of removing redundancies and fixing bugs is a great habit to have, especially when you have to do these sort of things when you're working as a developer